VideoSingle.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import WebView from "react-native-webview";
+import {
+  View,
+  Text,
+  StyleSheet,
+} from "react-native";
+
+  const formatDateString = (dateString) => {
+    const parts = dateString.split(" ")[0].split("-");
+    const [year, month, date] = parts;
+    return `${month} / ${date} / ${year}`;
+  };
+
+const VideoSingle = ({ event }) => {
+  return (
+    <>
+      <WebView
```

Comment:
Nice fix for adding Webview for the videos. Hopefully the thumb nails will fix it.